orc8r/cloud/deploy/terraform/orc8r-helm-aws/variables.tf

```diff
@@ -378,3 +378,20 @@ variable "analytics_category_name" {
   type = string
   default = "magma"
 }
+
```

Comment:
In this file there is an `orc8r_chart_version` variable set to 1.5.10. Can you update this to 1.5.11. Also update the `upgrade_1_4.md` at `magma/docs/orc8r/` to reference 1.5.11 instead of 1.5.10.